### PR TITLE
hipcc ignores --rocm-path if ROCM_PATH set

### PR DESCRIPTION
**Pull request by david-salinas**
swdev-343493 - hipcc ignores --rocm-path
    swdev-342125 - Fix ROCM_PATH issues

### CODE REVIEW

**Comment by scchan** [comment]
Could you please split it into 2 patches - one with the clean-up/refactoring and the other with the actual fix?  That would make the fix easier to follow

File: src/hipBin_base.h [hunk 1]
```diff
@@ -280,69 +283,73 @@ void HipBinBase::readEnvVariables() {
...
-    roccm_path = roccm_path.parent_path();
-    fs::path rocm_agent_enumerator_file(roccm_path);
-    rocm_agent_enumerator_file /= "bin/rocm_agent_enumerator";
-    if (!fs::exists(rocm_agent_enumerator_file)) {
-      roccm_path = "/opt/rocm";
-    }
-    variables_.roccmPathEnv_ = roccm_path.string();
-  } else {
-    variables_.roccmPathEnv_ = envVariables_.roccmPathEnv_;}
+	// we need to use --rocm-path option
```
Comment (scchan) [inline src/hipBin_base.h:337]: This function name should be corrected too?

File: src/hipBin_amd.h [hunk 2]
```diff
@@ -481,7 +483,7 @@ void HipBinAmd::checkHipconfig() {
 void HipBinAmd::printFull() {
   const string& hipVersion = getHipVersion();
   const string& hipPath = getHipPath();
-  const string& roccmPath = getRoccmPath();
+  const string& roccmPath = getRocmPath();
```
Comment (scchan) [inline src/hipBin_amd.h:486]: fix `roccmPath` name

**Review by scchan — CHANGES_REQUESTED** [review]
As mentioned in my previous comment, could you put the code renaming/refactoring changes in a separate PR because it's difficult to isolate and to reason the logic fixing the --rocm-path issue.

**Comment by david-salinas** [comment]
Abandoning this PR in favour of https://github.com/ROCm-Developer-Tools/HIPCC/pull/28

**Pull request closed without merging**